Model/login_screen.py

```diff
@@ -27,15 +27,22 @@ def data(self):
         """
         return self._data
 
-    @data.setter
+    @data.setter 
     def data(self, value):
         # We notify the View -
         # :class:`~View.ProfileScreen.profile_screen.ProfileScreenView` about the
         # changes that have occurred in the data model.
         self._data = value
         self.notify_observers("profile screen")
 
-    @multitasking.task
-    def check_data(self):
-        """Just an example of the method. Use your own code."""
-        self.data = ["example item"]
+    def is_account_taken(self, login_info):
+        """A method that checks if certain username and password exist in database"""
+        self.username = login_info[0]
+        self.password = login_info[1]
+        self.db = self.database.get_data()
+
+        for value in self.db.values(): 
+            if value["Username"] == self.username and value["Password"] == self.password:  
```

Comment:
Ditong part, question lang ako. Bakit need pa icheck if password is equal since username lang is enough para malaman na account is taken na diba?